src/Web/Static/documents/documentation/manual/v3/msbuild/daily_builds.html.md

```diff
@@ -30,4 +30,18 @@ The WixToolPath must be set to point to the location of the WiX tools directory
 
 You can also use a relative path to the directory (such as <strong>..\\..\\tools\\</strong>), but note that the WixTargetsPath property value must be relative to the .wixproj project file that uses it. The WixTasksPath property is used inside wix.targets to load WixTasks.dll; its value, if a relative path, must be relative to the wix.targets file. Those two files usually live together, so the value would be WixTasks.dll with no extra path information.
 
-**Note that WixToolPath must end in a backslash.**
+**Note that `WixToolPath` must end in a backslash.**
+
+### Custom Actions Projects
+
+For a custom action project, additional properties should be specified.  Open the `.csproj` in any text editor, such as Visual Studio, and add the following to the file anywhere between the `<Project>` element before the `<Import>` element:
+
+<pre><font size="2" color="#0000FF">&lt;<font size="2" color="#A31515">PropertyGroup</font>&gt;
+      &lt;</font><font size="2" color="#A31515">WixSdkPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">$(SourceCodeControlRoot)\wix\[[Version]]\sdk\</font><font size="2" color="#0000FF">\&lt;/</font><font size="2" color="#A31515">WixSdkPath</font><font size="2" color="#0000FF">&gt;
+      &lt;</font><font size="2" color="#A31515">WixCATargetsPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">$(WixSdkPath)wix.ca.targets</font><font size="2" color="#0000FF">&lt;/</font><font size="2" color="#A31515">WixCATargetsPath</font><font size="2" color="#0000FF">&gt;
+      &lt;</font><font size="2" color="#A31515">WixToolPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">$(SourceCodeControlRoot)\wix\[[Version]]\</font><font size="2" color="#0000FF">\&lt;/</font><font size="2" color="#A31515">WixToolPath</font><font size="2" color="#0000FF">&gt;
+      &lt;</font><font size="2" color="#A31515">WixTargetsPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">$(WixToolPath)Wix.targets</font><font size="2" color="#0000FF">&lt;/</font><font size="2" color="#A31515">WixTargetsPath</font><font size="2" color="#0000FF">&gt;
+      &lt;</font><font size="2" color="#A31515">WixTasksPath</font><font size="2" color="#0000FF">&gt;</font><font size="2">$(WixToolPath)wixtasks.dll</font><font size="2" color="#0000FF">&lt;/</font><font size="2" color="#A31515">WixTasksPath</font><font size="2" color="#0000FF">&gt;
+&lt;</font><font size="2" color="#A31515">/PropertyGroup</font><font size="2" color="#0000FF">&gt;</font></pre>
+
+Also, be certain to reference the local copy of the `Microsoft.Deployment.WindowsInstaller.dll` as opposed to the version installed in the GAC.
```

Comment:
Hmm, DTF should not be in the GAC. I don't think this comment makes sense.

Reply:
Perhaps I should change the verbiage.  I believe I had initially created the project from a template, and there was a reference to the dll in `C:\Program Files (x86)\WiX Toolset v3.11\SDK`.  I had to modify this to a local reference to get the project to build on the build server.  Would the phrase "... version installed on the development computer" or something similar be better?

Reply:
Yes, that verbiage would be better. We do not recommend nor condone GAC'ing WiX assemblies. :smile:

IIRC (and I don't use DTF myself), you can use the WIX environment var (like `$(Wix)\sdk\...`) to make the project more portable.

Reply:
Note that this file is autogenerated, the source for wix3 is at https://github.com/wixtoolset/wix3/blob/develop/src/chm/documents/msbuild/daily_builds.html.md.

I think that it should recommend using `$(WixSdkPath)` (or `$(WixToolsPath)`?) as the root for the reference to any WiX dlls.